### PR TITLE
Fix Dockerfile for development

### LINKED ISSUE #14: error in docker-compose
when I run the comand `$ docker-compose -f docker-compose.dev.yml up` the container start to install, but when it load the /.setup.py install, I got a "not found" error:
```
...
(30/30) Installing build-dependencies (20201007.144957)
Executing busybox-1.31.1-r16.trigger
OK: 358 MiB in 65 packages
Requirement already up-to-date: pip in /usr/local/lib/python3.7/site-packages (20.2.3)
Requirement already up-to-date: setuptools in /usr/local/lib/python3.7/site-packages (50.3.0)
/bin/sh: ./setup.py: not found
ERROR: Service 'api' failed to build : The command '/bin/sh -c apk update && apk --no-cache add --virtual build-dependencies     
gcc     musl-dev     postgresql-dev     tzdata &&    pip install -U pip setuptools &&    ./setup.py install &&    api collectstatic &&    rm -rf /opt/acompanha_legis/*' returned a non-zero code: 127
```

Did I miss something?

### LINKED ISSUE #17: Split dockerfile for prod and dev
The `api/Dockerfile` is for prod, it is better to split it.

### PR DESCRIPTION
**Pull request by kimuraz**
This also fixes some issues that Windows users were having to get the project up and running. Besides that, it improves the structure of the fields by making the docker-compose file for development the default one.

Fix #14

### CODE REVIEW

**Comment by kimuraz** [comment]
@BrunaNayara Se os acentos estiverem "encoded" errados, você pode consertar? Fiz esses commits no windows e o powershell + vim é estranho.

File: README-ptbr.md [hunk 1]
```diff
@@ -41,12 +41,22 @@ Queremos ter as instruções de instalação local, mas ainda não temos.
...
-$ docker-compose -f docker-compose.dev.yml up
+$ docker-compose up
 ```
 Se quiser deixar o projeto rodando sem ocupar o seu terminal, basta adicionar a flag `-d` ao final do comando acima.
 
 Depois abra `http://localhost:8000/` no seu navegador
 
+## Problemas
+
+Para algumas pessoas que fizeram o "build" do projeto anteriorment (docker-compose.dev.yml) vocêpode enfrentar algums problemas, tente os seguintes comandos:
```
Comment (BrunaNayara) [inline README-ptbr.md]:
```suggestion
Para algumas pessoas que fizeram o "build" do projeto anteriorment (docker-compose.dev.yml) você pode enfrentar algums problemas, tente os seguintes comandos:
```

File: README-ptbr.md [hunk 2]
```diff
@@ -41,12 +41,22 @@ Queremos ter as instruções de instalação local, mas ainda não temos.
...
+
+````
+$ docker-compose down && docker-compose build --no-cache
+````
+
+> :warning: Se problema persistir por favor abra uma issue para conseguir ajudar, voc� pode tentar remover manualmente as imagens api e task e rodar novamente `docker-compose up`
```
Comment (BrunaNayara) [inline README-ptbr.md:58]:
```suggestion
> :warning: Se problema persistir por favor abra uma issue para tentarmos ajudar. Você pode tentar remover manualmente as imagens api e task e rodar novamente `docker-compose up` 
```

**Review by BrunaNayara — DISMISSED** [review]

File: README-ptbr.md (hunk 2 above)
Comment (BrunaNayara) [inline README-ptbr.md]:
```suggestion
> :warning: Se problema persistir por favor abra uma issue para conseguir ajudar, você pode tentar remover manualmente as imagens api e task e rodar novamente `docker-compose up` 
```

Comment (BrunaNayara) [inline README-ptbr.md]:
```suggestion
> :warning: Se problema persistir por favor abra uma issue para conseguir ajudar, você pode tentar remover manualmente as imagens api e task e rodar novamente `docker-compose up` 
```

File: README-ptbr.md [hunk 3]
```diff
@@ -41,13 +41,24 @@ Queremos ter as instruções de instalação local, mas ainda não temos.
...
+$ docker-compose up
 ```
 Se quiser deixar o projeto rodando sem ocupar o seu terminal, basta adicionar a flag `-d` ao final do comando acima.
 
 Depois abra `http://localhost:8000/` no seu navegador
 
 
+## Problemas
+
+Para algumas pessoas que fizeram o "build" do projeto anteriorment (docker-compose.dev.yml) você pode enfrentar algums problemas, tente os seguintes comandos:
```
Comment (BrunaNayara) [inline README-ptbr.md]:
```suggestion
Para algumas pessoas que fizeram o "build" do projeto anteriormente (docker-compose.dev.yml) você pode enfrentar algums problemas, tente os seguintes comandos:
```

**Review by BrunaNayara — APPROVED** [review]

**Pull request merged**